docs/react-testing-library/setup.md

```diff
@@ -328,3 +273,13 @@ mocha --require jsdom-global/register
 Note, depending on the version of Node you're running, you may also need to
 install @babel/polyfill (if you're using babel 7) or babel-polyfill (for babel
 6).
+
+### Skipping Auto Cleanup
+
+[`Cleanup`](./api#cleanup) is called after each test automatically by default.
+However, you may choose to skip the auto cleanup by add setting the
+RTL_SKIP_AUTO_CLEANUP env variable to 'true'.
+
+```javascript
+RTL_SKIP_AUTO_CLEANUP = 'true'
```

Comment:
Let's add a reference to [`cross-env`](https://github.com/kentcdodds/cross-env) here and suggest that they can put this in their test script.

Then this line can be:

```suggestion
cross-env RTL_SKIP_AUTO_CLEANUP=true jest
```

Reply:
Okay. How would you word that?

Reply:
Can you help me out with the wording? Not sure how this would be done, so it will be hard to suggest :)